project1/index.html

```diff
@@ -0,0 +1,175 @@
+        rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css?family=Cute+Font|Lato:300,300i,400,400i,700,700i,900,900i|Roboto:300i,400,400i,500i,700,700i,900&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Prata&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    <main>
+        <h1>
+            The Crane Wife <a href=https://longform.org/posts/the-crane-wife> </a> </h1> <h2>By
+                <Cite>CJ Hauser</cite></h2>
+                <h3>July 16, 2019</h3>
+
+
+
+                <p>Ten days after I called off my engagement I was supposed to go on a scientific expedition to
+                    study the
+                    whooping
+                    crane on the gulf coast of Texas. <em>Surely,</em> I will cancel this trip, I thought, as I
+                    shopped for nylon
+                    hiking
+                    pants that zipped off at the knee. <em>Surely,</em> a person who calls off a wedding is meant to
+                    be sitting
+                    sadly at
+                    home,
+                    reflecting on the enormity of what has transpired and not doing whatever it is I am about to be
+                    doing that
+                    requires a pair of plastic clogs with drainage holes. <em>Surely,</em> I thought, as I tried on a
+                    very
+                    large and
+                    floppy
+                    hat
+                    featuring a pull cord that fastened beneath my chin, <i>it would be wrong to even be wearing a
+                        hat that looks
+                        like
+                        this when
+                        something in my life has gone so terribly wrong.</i>
+
+                    Ten days earlier <strong>I had cried and I had yelled and I had packed up
+                        my dog and driven away from
+                        the upstate
+                        New
+                        York house</strong> with two willow trees I had bought with my fiancé.
+
+                    Ten days later and I didn’t want to do anything I was supposed to do.
+                </p>
+
+
+
+
+                <p>
+                    I went to Texas to study the whooping
+                    crane
+                    because I was researching a
+                    novel. In my
+                    novel there were
+                    biologists
+                    doing field research about birds and I had no idea what field research actually
+                    looked
+                    like and so the
+                    scientists in
+                    my novel draft did things like shuffle around great stacks of papers and frown.
+                    The
+                    good people of
+
+                    <i><a href=https://earthwatch.org>the Earthwatch organization</a> </i> assured me I was welcome on
```

Comment:
The `href` value needs to be wrapped in double quotes, like so:
```
<a href="https://earthwatch.org">the Earthwatch organization</a>
```
But, otherwise, this was a well-formed anchor element! 👍🏻